uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -414,6 +548,31 @@ define([
     }
 
     /**
+     * Loads and sets time ranges from work pattern for "from" or "to" timepickers
+     *
+     * @param  {Date} date in standard JavaScript format
+     * @param  {String} type (days|hours)
+     * @return {Promise}
+     */
+    function loadAndSetTimeRangesFromWorkPattern (date, type) {
+      var timeObject = vm.uiOptions.times[type];
+
+      if (!date) { return $q.resolve(); }
+
+      timeObject.disabled = true;
+      timeObject.time = '';
+
+      return vm.request.getWorkDayForDate(date).then(function (response) {
+        timeObject.min = response.time_from || '00:00';
```

Comment:
aren't these set by default by the directive (00:00 and 23:45) ?  Why repeat these defaults here?